apps/api-extractor/src/analyzer/AstImport.ts

```diff
@@ -32,18 +54,12 @@ export class AstImport {
   public readonly modulePath: string;
 
   /**
-   * If modulePath is defined, then this specifies the export name for the definition.
-   *
-   * Example: "IBuildConfig"
+   * For `AstImportKind.Normal`, returns `X` from `import { X } from "y";`.
```

Comment:
The quoting in this comment is weird. Why not just follow our standards. Code-fence everything and use comments for commentary?

Reply:
Fixed